docs/outlook/outlook-mobile-addins.md

```diff
@@ -27,6 +27,8 @@ Outlook mobile add-ins are supported on all Microsoft 365 business accounts and
   - The add-in **MUST** adhere to the [UI guidelines](outlook-addin-design.md).
   - The scenario for the add-in **MUST** [make sense on mobile](#what-makes-a-good-scenario-for-mobile-add-ins).
 
+    [!INCLUDE [Teams manifest not supported on mobile devices](../includes/no-mobile-with-json-note.md)]
```

Comment:
I feel like we should move this note to the top of the section (right after the title) OR remove the indent. In this location and indented like that, it seems it would somehow apply to these bullets above it which isn't how I understand this note.
```suggestion
[!INCLUDE [Teams manifest not supported on mobile devices](../includes/no-mobile-with-json-note.md)]
```